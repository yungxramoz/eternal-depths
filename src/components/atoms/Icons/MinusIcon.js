import React from 'react'
import './Icon.css'

const MinusIcon = () => {
  return (
    <svg
      fill="currentColor"
      id="Minus"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 24 24"
    >
      <rect x="1" y="11" width="22" height="2" />
    </svg>
  )
}

export default MinusIcon
