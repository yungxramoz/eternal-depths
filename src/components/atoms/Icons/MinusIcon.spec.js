import React from 'react'
import TestRenderer from 'react-test-renderer'
import MinusIcon from './MinusIcon'

describe('MinusIcon', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<MinusIcon />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
