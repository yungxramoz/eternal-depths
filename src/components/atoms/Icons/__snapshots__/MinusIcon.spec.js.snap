// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinusIcon renders correctly: MinusIcon 1`] = `
<body>
  <div>
    <svg
      fill="currentColor"
      id="Minus"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <rect
        height="2"
        width="22"
        x="1"
        y="11"
      />
    </svg>
  </div>
</body>
`;
